server/game/cards/13-O4B/BayouVermilionRailroad.js

```diff
@@ -0,0 +1,39 @@
+const OutfitCard = require('../../outfitcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class BayouVermilionRailroad extends OutfitCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'current',
+            effect: ability.effects.reduceFirstPlayedCardCostEachRound(1, card => card.hasKeyword('mystical'))
+        });
+        this.action({
+            title: 'Bayou Vermilion Railroad',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),        
+            target: {
+                activePromptTitle: 'Select a Mystical card',
+                cardCondition: { 
+                    location: 'play area',
+                    condition: card => card.hasKeyword('mystical') && card.isAtDeed() && !card.booted && card.canBeBooted()
+                }
+            },
+            handler: context => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: context.target }), context);
+                context.player.modifyGhostRock(1);
+                this.untilEndOfPhase(context.ability, ability => ({
+                    match: context.target.locationCard,
+                    effect: [
+                        ability.effects.setProduction(0),
+                        ability.effects.setControl(0)
+                    ]
+                }), 'upkeep');
```

Comment:
> you may reduce the deed's control points and production to 0 until after the next Upkeep phase

It says you may reduce, so we should put a prompt to ask player if he wants to reduce.

Reply:
good catch, i'll wrap it into a prompt tonight.

Reply:
I added the prompt and some game messages.